app/javascript/components/projects/ProjectsIndex.jsx

```diff
@@ -207,11 +208,14 @@ class ProjectsIndex extends React.Component {
                             </div>
                         </div>
                     </div>
-                    <a
+                    {
+                      user_type === 'User' ? '' :
```

Comment:
Can we check strictly for `Organization` as a user type here instead? And all others don't see a Create Project button?

Reply:
Makes sense since only `Organization` (Project Leads) will be the only ones that have access to that -- i'll make that change!

Reply:
doonneee @ethanlee16 thank you for pointing that out!